packages/nds-stacked-nav/stacked-nav.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "@nice-digital/nds-stacked-nav" {
+	import React = require("react");
+
+	export interface StackedNavLinkProps {
+		destination?: string;
+		isCurrent?: boolean;
+		label?: React.ReactNode;
+		elementType?: React.ElementType | string;
+		hint?: string;
+		children?: string;
+	}
+
+	export interface StackedNavProps {
```

Comment:
I think this is missing the children:

```
children: React.ReactElement<StackedNavLink>[] | React.ReactElement<StackedNavLink>;
```

or whatever the syntax is

Reply:
Also the same comment applies as per the StackedNavLink - can we do some TS jiggery pokery to require either label or children?